src/waitandsee.jl

```diff
@@ -1,7 +1,7 @@
 export waitandsee
 
-mutable struct WSPath
-    node::SDDPNode
+mutable struct WSPath{NodeT}
```

Comment:
Wouldn't it be clearer to write `WaitAndSeePath`? I tend to think that explicit is always better than implicit^^

Reply:
Done :)